src/components/biggive-cookie-banner/biggive-cookie-banner.tsx

```diff
@@ -0,0 +1,57 @@
+import { Component, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'biggive-cookie-banner',
+  styleUrl: 'biggive-cookie-banner.scss',
+  shadow: true,
+})
+export class BiggiveCookieBanner {
+  @Prop() blogUriPrefix: string;
+
+  render() {
+    return (
+      <div class="cooke-consent-container">
+        <h2>Our website uses cookies</h2>
+
+        <p>
+          We use some essential cookies to make our website work. We'd also like to use analytics cookies to understand how you use the website, make improvements and share data
```

Comment:
I think this should probably be flipped so as to be clearer that the sharing is not through cookies.

Reply:
Right. Done in daa4c0c.